src/App.js

```diff
@@ -1,12 +1,14 @@
 import React from "react"
 import Navbar from "./components/Navbar"
+import SubHeader from "./components/SubHeader"
 import './App.css';
+import './subHeader.css';
```

Comment:
This should be within the component's folder
```
Components/
|___SubHeader/
        |___SubHeader.js
        |___SubHeader.css
 ```
 - import the `subHeader.css` in `SubHeader.js`